src/msg/async/ProtocolV2.cc

```diff
@@ -1351,7 +1352,6 @@ CtPtr ProtocolV2::handle_message() {
 #if defined(WITH_EVENTTRACE)
```

Comment:
this should probably get moved too?

Reply:
Agreed and modified.
better to make it aligned with V1, so we don't need to introduce a new ltt_recv_stamp just for LTT.